### ISSUE TITLE
Do not allow continuing/suggesting with invalid/archived data

### ISSUE THREAD

**Issue by amulware** [toggl/mobileapp#1344]
See  #1337.

This may already be partly or fully implemented, but let's make 100% sure we're getting this right, and maybe even see if we can somehow enforce this for good.

**Comment by amulware** [comment]
We will allow continuing TEs with archived projects, however the new TE should then have no project assigned. I'll make a new issue for this, since we might want special UI in this case (a toast letting the user know what happened maybe).

**Comment by simonrozsival** [comment]
For now it doesn't really matter - there aren't any project which would be marked as archived - we should work on this after #1338 is finished and there are we start having actual archived projects in the DB.
